### PR TITLE
v2.19.8

### PR DESCRIPTION
**Pull request by allelos**
Netdata dashboard version 2.19.8

### CODE REVIEW

**Review by Ferroin — APPROVED** [review]

**Review by vkalintiris — APPROVED** [review]

File: web/gui/dashboard/tv-react.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata TV Dashboard</title>
```
Comment (ilyam8) [inline web/gui/dashboard/tv-react.html:5]: @andrewm4894 👀

File: web/gui/dashboard/tv-react.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+    <title>NetData TV Dashboard</title>
```
Comment (ilyam8) [inline web/gui/dashboard/tv-react.html:5]: @andrewm4894 👀

Comment (andrewm4894) [inline web/gui/dashboard/tv-react.html:5]:
```suggestion
    <title>Netdata TV Dashboard</title>
```

Comment (andrewm4894) [reply web/gui/dashboard/tv-react.html:5]: @allelos howdy - ok to use "Netdata" as opposed to "NetData"?

File: web/gui/dashboard/infographic.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <html lang=en-us>
 <head>
     <meta charset=utf-8>
-    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
+    <title>NetData: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/infographic.html:6]:
```suggestion
    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```

File: web/gui/dashboard/index-node-view.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-  <title>Netdata TV Dashboard</title>
+  <title>NetData TV Dashboard</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/index-node-view.html:5]:
```suggestion
  <title>Netdata TV Dashboard</title>
```

File: web/gui/dashboard/demosites2.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata - Real-time performance monitoring, done right!</title>
+    <title>NetData - Real-time performance monitoring, done right!</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/demosites2.html:5]:
```suggestion
    <title>Netdata - Real-time performance monitoring, done right!</title>
```

File: web/gui/dashboard/demosites.html [hunk 6]
```diff
@@ -4,7 +4,7 @@
 <head>
     <meta http-equiv="Refresh" content="0; url=https://www.netdata.cloud">
     <meta charset=utf-8>
-    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
+    <title>NetData: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/demosites.html:7]:
```suggestion
    <title>Netdata: Get control of your Linux Servers. Simple. Effective. Awesome.</title>
```

File: web/gui/dashboard/demo2.html [hunk 7]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata Dashboard</title>
+    <title>NetData Dashboard</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/demo2.html:5]:
```suggestion
    <title>Netdata Dashboard</title>
```

File: web/gui/dashboard/demo.html [hunk 8]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata Dashboard</title>
+    <title>NetData Dashboard</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/demo.html:5]:
```suggestion
    <title>Netdata Dashboard</title>
```

File: web/gui/dashboard/dashboard.html [hunk 9]
```diff
@@ -518,8 +518,8 @@ <h1>Gauge.js</h1>
 
 <hr>
 <h1>Google Charts</h1>
-Netdata was originaly developed with Google Charts.
-Netdata is a complete Google Visualization API provider.
+NetData was originaly developed with Google Charts.
```
Comment (andrewm4894) [inline web/gui/dashboard/dashboard.html:521]:
```suggestion
Netdata was originaly developed with Google Charts.
```

File: web/gui/dashboard/dashboard.html [hunk 10]
```diff
@@ -518,8 +518,8 @@ <h1>Gauge.js</h1>
...
+NetData is a complete Google Visualization API provider.
```
Comment (andrewm4894) [inline web/gui/dashboard/dashboard.html:522]:
```suggestion
Netdata is a complete Google Visualization API provider.
```

File: web/gui/dashboard/dashboard.html [hunk 11]
```diff
@@ -25,7 +25,7 @@
 
 <div class="container-fluid">
 
-<h1>Netdata Custom Dashboard <div data-netdata="system.cpu" data-chart-library="sparkline" data-height="30" data-after="-600" data-sparkline-linecolor="#888"></div></h1>
+<h1>NetData Custom Dashboard <div data-netdata="system.cpu" data-chart-library="sparkline" data-height="30" data-after="-600" data-sparkline-linecolor="#888"></div></h1>
```
Comment (andrewm4894) [inline web/gui/dashboard/dashboard.html:28]:
```suggestion
<h1>Netdata Custom Dashboard <div data-netdata="system.cpu" data-chart-library="sparkline" data-height="30" data-after="-600" data-sparkline-linecolor="#888"></div></h1>
```

File: web/gui/dashboard/dashboard.html [hunk 12]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata Dashboard</title>
+    <title>NetData Dashboard</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/dashboard.html:5]:
```suggestion
    <title>Netdata Dashboard</title>
```

File: web/gui/dashboard/tv.html [hunk 13]
```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: GPL-3.0-or-later -->
 <html lang="en">
 <head>
-    <title>Netdata TV Dashboard</title>
+    <title>NetData TV Dashboard</title>
```
Comment (andrewm4894) [inline web/gui/dashboard/tv.html:5]:
```suggestion
    <title>Netdata TV Dashboard</title>
```

**Comment by andrewm4894** [comment]
I added a few suggestions to just use "Netdata" instead of the old "NetData" which i had fixed here: https://github.com/netdata/netdata/pull/11484

Just did'nt want to commit those small suggestions here myself but should be just small easy ones.

**Comment by allelos** [comment]
> I added a few suggestions to just use "Netdata" instead of the old "NetData" which i had fixed here: #11484
> 
> Just did'nt want to commit those small suggestions here myself but should be just small easy ones.

@andrewm4894 There's no value in changing them here, since they will be overwritten with every dashboard release. These have to be changed on the dashboard repo.

**Comment by ilyam8** [comment]
> These have to be changed on the dashboard repo.

The PR is https://github.com/netdata/dashboard/pull/303

**Review by ilyam8 — APPROVED** [review]

**Comment by allelos** [comment]
I'm closing this since changes from 2.19.8 alongside with `NetData -> Netdata` will be on `2.19.9` release.

**Pull request closed without merging**